src/serac/physics/utilities/weak_form/host_device_annotations.hpp

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
```

Comment:
This should probably go into infrastructure.

Reply:
Can probably go in `accelerator.hpp`.